src/AppBundle/Service/Mailer/MailSenderMock.php

```diff
@@ -3,44 +3,70 @@
 namespace AppBundle\Service\Mailer;
 
 use AppBundle\Model\Email;
+use Predis\Client as PredisClient;
 use Swift_Attachment;
+use Swift_Mailer;
 use Swift_Message;
-use Symfony\Bundle\FrameworkBundle\Translation\Translator;
-use Symfony\Component\DependencyInjection\Container;
-use Symfony\Component\Validator\Validator;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
 
 class MailSenderMock extends MailSender
```

Comment:
I'm not a fan of this class extending the `MailSender`, as it's not a good fit for inheritance. I think we need a `MailSenderInterface` and both `MailSender` and `MailSenderMock` become implementations of that.